src/platform/intel/cavs/ext_manifest.c

```diff
@@ -10,15 +10,17 @@
 #include <sof/common.h>
 #include <sof/lib/memory.h>
 
+/* increment this value after adding any element to ext_man_cavs_config dictionary */
+#define CAVS_CONFIG_ELEM_CNT 3
+
 const struct ext_man_cavs_config_data ext_man_cavs_config
 	__aligned(EXT_MAN_ALIGN) __section(".fw_metadata") = {
 	.hdr.type = EXT_MAN_ELEM_PLATFORM_CONFIG_DATA,
-	.hdr.elem_size = ALIGN_UP(sizeof(struct ext_man_cavs_config_data),
+	.hdr.elem_size = ALIGN_UP(sizeof(struct ext_man_cavs_config_data) +
+				  sizeof(struct config_elem) * CAVS_CONFIG_ELEM_CNT,
```

Comment:
Same suggestion above.

Reply:
@ktrzcinx @xiulipan  any comments - are these now resolved ? Btw CI is passing.